.github/workflows/cd.yaml

```diff
@@ -1,11 +1,11 @@
 name: CD
 
 on:  # yamllint disable-line rule:truthy
-  create:
-    tags: ["v?[0-9]+.[0-9]+.[0-9]+"]
+  create
 
 jobs:
   publish:
+    if: ${{ startsWith(github.ref, 'refs/tags/v') }}
```

Comment:
Should this be `'refs/tags'` instead of `'refs/tags/v'` since our tags currently do not start with `v` (ex: [0.54.1](https://github.com/zappa/Zappa/releases/tag/0.54.1))?

FWIW, the [`pypa/gh-action-pypi-publish` documentation uses this in their examples](https://github.com/pypa/gh-action-pypi-publish#usage):

`if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')`

Should we use that as well?  I'm not sure of the significance of excluding `github.event_name == 'push'`?